lib/get-vimeo-url.js

```diff
@@ -6,7 +6,13 @@ function getVimeoUrl(videoId, cb) {
   Request({
     url: 'https://player.vimeo.com/video/' + videoId + '/config',
     onComplete: function (resp) {
-      cb(null, resp.json.request.files.progressive[0].url);
+      if (!resp.json) cb('Could not connect to vimeo.com');
```

Comment:
I'm a bit confused here.
Something feels 🐟 since we aren't doing a `return` here, so we could technically call the callback twice (here and on line 14 below).
Or more likely we'd implode on L14 below since `resp.json` would be falsey here.